QtCollider/QObjectProxy.cpp

```diff
@@ -54,7 +54,6 @@ QObjectProxy::QObjectProxy( QObject *qObject_, PyrObject *scObject_ )
   _scObject( scObject_ ),
   _scClassName( slotRawSymbol( &scObject_->classptr->name )->name )
 {
-  ProxyToken *token = new ProxyToken( this, qObject );
```

Comment:
this might require more discussion/analysis, you may want to leave it in for now (many of the other changes are easy to validate)